xarray/core/variable.py

```diff
@@ -28,14 +28,26 @@
     pass
 
 
-def as_variable(obj, name=None, copy=False):
-    """Convert an object into an Variable.
-
-    - If the object is already an `Variable`, return a shallow copy.
-    - Otherwise, if the object has 'dims' and 'data' attributes, convert
-      it into a new `Variable`.
-    - If all else fails, attempt to convert the object into an `Variable` by
-      unpacking it into the arguments for `Variable.__init__`.
+def as_variable(obj, name=None):
+    """Convert an object into a Variable.
+
+    Parameters
+    ----------
+    obj : object
+        Object to convert into a Variable.
+        If the object is already a Variable, return a shallow copy.
```

Comment:
Can we keep these as bullet points for sphinx? I think that works for rendered docstrings if you're careful (e.g., see the [docstring for `xarray.Dataset.__init__`](http://xarray.pydata.org/en/stable/generated/xarray.Dataset.html#xarray.Dataset)).